docs/book/component-gallery/data-validators/deepchecks.md

```diff
@@ -249,7 +249,7 @@ pipeline = data_validation_pipeline(
 pipeline.run()
 ```
 
-As can be seen from the [step definition](https://apidocs.zenml.io/latest/api_docs/integration_code_docs/integrations-deepchecks/#zenml.integrations.deepchecks.steps.deepchecks_data_integrity.DeepchecksDataIntegrityCheckStepConfig),
+As can be seen from the [step definition](https://apidocs.zenml.io/0.20.5/integration_code_docs/integrations-deepchecks/#zenml.integrations.deepchecks.steps.deepchecks_data_integrity.DeepchecksDataIntegrityCheckStepParameters),
```

Comment:
Keep it at latest please. It should be `https://apidocs.zenml.io/latest/integration_code_docs/integrations-deepchecks/#zenml.integrations.deepchecks.steps.deepchecks_data_integrity.DeepchecksDataIntegrityCheckStepParameters`

Reply:
Done, though the problem we had was that latest no longer made sense since that code didn't exist any more.